Handy/Handy-Storybook/Atom/ChipViewController.swift

```diff
@@ -0,0 +1,143 @@
+        chip.addTarget(self, action: #selector(firstChip), for: .touchUpInside)
+        chip.translatesAutoresizingMaskIntoConstraints = false
+        return chip
+    }()
+    
+    let unTappedSmallChip: HandyChip = {
+        let chip = HandyChip()
+        chip.chipState = HandyChip.ChipState(type: .enabled)
+        chip.size = HandyChip.ChipSize(type: .small)
+        chip.isSelected = false
+        chip.text = "Label"
+        chip.leftIcon = UIImage(named: "icInfoCircle")
+        chip.rightIcon = UIImage(named: "icClose")
+        chip.addTarget(self, action: #selector(secondChip), for: .touchUpInside)
+        chip.translatesAutoresizingMaskIntoConstraints = false
+        return chip
+    }()
+    
+    let tappedLargeChip: HandyChip = {
+        let chip = HandyChip()
+        chip.chipState = HandyChip.ChipState(type: .enabled)
+        chip.size = HandyChip.ChipSize(type: .large)
+        chip.isSelected = true
+        chip.text = "Label"
+        chip.leftIcon = UIImage(named: "icInfoCircle")
+        chip.rightIcon = UIImage(named: "icClose")
+        chip.addTarget(self, action: #selector(thirdChip), for: .touchUpInside)
+        chip.translatesAutoresizingMaskIntoConstraints = false
+        return chip
+    }()
+    
+    let tappedSmallChip: HandyChip = {
+        let chip = HandyChip()
+        chip.chipState = HandyChip.ChipState(type: .enabled)
+        chip.size = HandyChip.ChipSize(type: .small)
+        chip.isSelected = true
+        chip.text = "Label"
+        chip.leftIcon = UIImage(named: "icInfoCircle")
+        chip.rightIcon = UIImage(named: "icClose")
+        chip.addTarget(self, action: #selector(fourthChip), for: .touchUpInside)
+        chip.translatesAutoresizingMaskIntoConstraints = false
+        return chip
+    }()
+    
+    let disabledLargeChip: HandyChip = {
+        let chip = HandyChip()
+        chip.chipState = HandyChip.ChipState(type: .disabled)
+        chip.size = HandyChip.ChipSize(type: .large)
+        chip.text = "Label"
+        chip.leftIcon = UIImage(named: "icInfoCircle")
+        chip.rightIcon = UIImage(named: "icClose")
+        chip.translatesAutoresizingMaskIntoConstraints = false
+        return chip
+    }()
+    
+    let disabledSmallChip: HandyChip = {
+        let chip = HandyChip()
+        chip.chipState = HandyChip.ChipState(type: .disabled)
+        chip.size = HandyChip.ChipSize(type: .small)
+        chip.text = "Label"
+        chip.leftIcon = UIImage(named: "icInfoCircle")
+        chip.rightIcon = UIImage(named: "icClose")
+        chip.translatesAutoresizingMaskIntoConstraints = false
+        return chip
+    }()
+    
+    
+    //MARK: Chip 뷰에 추가
+    override func setViewHierarchies() {
+        view.addSubview(unTappedLargeChip)
+        view.addSubview(unTappedSmallChip)
+        view.addSubview(tappedLargeChip)
+        view.addSubview(tappedSmallChip)
+        view.addSubview(disabledLargeChip)
+        view.addSubview(disabledSmallChip)
+    }
```

Comment:
```suggestion
    override func setViewHierarchies() {
        [unTappedLargeChip, 
          unTappedSmallChip, 
          tappedLargeChip, 
          tappedSmallChip, 
          disabledLargeChip, 
          disabledSmallChip
        ].forEach {
            view.addSubview($0)
        }
    }
```
  view.addSubviews(unTappedLargeChip, unTappedSmallChip, tappedLargeChip, tappedSmallChip, disabledLargeChip, disabledSmallChip) 이런식으로도 사용하실 수 있어요~